pkgs/applications/audio/guitarix/default.nix

```diff
@@ -22,15 +23,14 @@ stdenv.mkDerivation rec {
   buildInputs = [
     avahi bluez boost eigen fftw glib glibmm gtk gtkmm jack2
     ladspaH librdf libsndfile lilv lv2 serd sord sratom
+    zita-convolver zita-resampler
   ];
 
   configureFlags = [
     "--shared-lib"
     "--no-desktop-update"
-    "--no-faust" # Need to package a release of faust, 0.9.58 or 0.9.65
     "--enable-nls"
-    "--includeresampler" # Zita-resampler not packaged, use vendored version
-    "--includeconvolver" # Zita-convolver not packaged, use vendored version
+    "--no-faust" # todo: find out why --faust doesn't work
```

Comment:
Faust doesn't work because only a git dev version is packaged, not an official release that guitarix works with, my previous comment (`# Need to package a release of faust, 0.9.58 or 0.9.65`) is still correct.